src/k8s-engine/k3sHelper.ts

```diff
@@ -7,11 +7,14 @@ import stream from 'stream';
 import tls from 'tls';
 import util from 'util';
 
+import dns from 'dns';
 import semver from 'semver';
 import { CustomObjectsApi, KubeConfig, V1ObjectMeta } from '@kubernetes/client-node';
 import { ActionOnInvalid } from '@kubernetes/client-node/dist/config_types';
+import { Response } from 'node-fetch';
 import yaml from 'yaml';
 
+import Electron from 'electron';
```

Comment:
We can do `import type Electron` here to ensure we don't end up using it from this code.

Reply:
done